### PR TITLE
Accepted revisions for SDL-0267 Main Menu UI Updates

### PR DESCRIPTION
**Pull request by joeljfischer**
### Updates
* Moved `AddCommand` text parameters into `MenuParams`.
* Made `submenu` into `subMenu` for consistency with the MOBILE_API.

### CODE REVIEW

**Comment by joeljfischer** [comment]
@theresalech this is ready for review

File: proposals/0267-addcommand-ui-updates.md [hunk 1]
```diff
@@ -65,16 +69,16 @@ We also need to expand our capabilities so that developers and high-level manage
...
+        <description>Secondary text for AddCommand</description>
     </element>
     <element name="menuCommandTertiaryText" since="x.x">
-        <description> Tertiary text for AddCommand</description>
+        <description>Tertiary text for AddCommand</description>
     </element>
-    <element name="menuSubmenuSecondaryText" since="x.x">
-        <description> Secondary text for AddSubmenu</description>
+    <element name="menuSubMenuSecondaryText" since="x.x">
+        <description>Secondary text for AddSubmenu</description>
```
Comment (theresalech) [inline proposals/0267-addcommand-ui-updates.md]:
```suggestion
        <description>Secondary text for AddSubMenu</description>
```

File: proposals/0267-addcommand-ui-updates.md [hunk 2]
```diff
@@ -65,16 +69,16 @@ We also need to expand our capabilities so that developers and high-level manage
...
     </element>
-    <element name="menuSubmenuTertiaryText" since="x.x">
-        <description> Tertiary text for AddSubmenu</description>
+    <element name="menuSubMenuTertiaryText" since="x.x">
+        <description>Tertiary text for AddSubmenu</description>
```
Comment (theresalech) [inline proposals/0267-addcommand-ui-updates.md]:
```suggestion
        <description>Tertiary text for AddSubMenu</description>
```

File: proposals/0267-addcommand-ui-updates.md [hunk 3]
```diff
@@ -86,7 +90,7 @@ We also need to expand our capabilities so that developers and high-level manage
     <element name="menuCommandSecondaryImage" since = "x.x">
         <description>The secondary image field for AddCommand</description>
     </element>
-    <element name="menuSubmenuSecondaryImage" since = "x.x">
+    <element name="menuSubMenuSecondaryImage" since = "x.x">
         <description>The secondary image field for AddSubmenu</description>
```
Comment (theresalech) [inline proposals/0267-addcommand-ui-updates.md]:
```suggestion
        <description>The secondary image field for AddSubMenu</description>
```

**Comment by joeljfischer** [comment]
@theresalech Yes, thanks for the catch. Everything has been addressed.

**Pull request merged**